formats/contact/publisher/details.json

```diff
@@ -0,0 +1,330 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "title",
+    "description"
+  ],
+  "properties": {
+    "slug": {
+      "type": "string"
+    },
+    "title": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "organisation": {
```

Comment:
We have, however, added `organisation` to the `links` hash in the example and schema, but kept this way around as well so we have a current view on the state of the app.